[OSD-15879] backplane support login to service cluster directly (#80)
openshift/backplane-cli: README.md

  $ oc whoami
  system:serviceaccount:openshift-backplane-srep:1234567
  ```
  
- To login to the mgmt cluster for hypershift or the hive cluster for normal OSD/Rosa
  ```
  $ ocm backplane login <cluster> --manager
  ```

> - To login to the service cluster for hypershift hosted cluster or the mgmt cluster

Reviewer: ```suggestion
- To login to the Service Cluster of a HyperShift hosted cluster or the Management Cluster
```
